script/utils/revapi-clean.sh

```diff
@@ -1,4 +1,19 @@
 #!/bin/sh
+# How to clean ignores in revapi-config.json files:
+#
+# export REVAPI_REPOSITORIES=droolsjbpm-build-bootstrap,droolsjbpm-knowledge,drools,jbpm,droolsjbpm-integration,optaplanner
+#
+# ./git-clone-others.sh --repo-list=$REVAPI_REPOSITORIES --add-upstream-remote
+# ./git-all.sh --repo-list=$REVAPI_REPOSITORIES checkout master
+# ./git-all.sh --repo-list=$REVAPI_REPOSITORIES pull --rebase upstream master
+# ./git-all.sh --repo-list=$REVAPI_REPOSITORIES checkout -b revapi-7.23.0.Final
+# Copy this script to root of all repositories and run it there, ./revapi-clean.sh 7.23.0.Final
+# Update <revapi.oldKieVersion> in kie-parent
+# ./mvn-all.sh --repo-list=$REVAPI_REPOSITORIES clean install -DskipTests
+# ./git-all.sh --repo-list=$REVAPI_REPOSITORIES add -u
+# ./git-all.sh --repo-list=$REVAPI_REPOSITORIES commit -m "BAQE-1039 - Change revapi to check against 7.23.0.Final"
+# ./git-all.sh --repo-list=$REVAPI_REPOSITORIES push origin revapi-7.23.0.Final
+# Create PRs
 # Utility script for removing ignores in revapi-config.json files.
 
```

Comment:
so why all lines are commented if this script helps only as a guide/doc?

Reply:
These are the steps for now (I can make a separate README.md file for revapi if you want) which help to clean revapi ignores semi-automatically. The revapi-clean.sh script is used in one of these steps, specifically on the line 10.

Reply:
I meant if it is script you probably want to uncomment these lines to be executed.

Reply:
I see your point now. No, these are all steps and the whole file `revapi-clean.sh` is just used for step on the line 10. Otherwise you just copy all commands on command line and execute them one after another.